### PR TITLE
Allow AMY to import an event using the GH repo URL.

### LINKED ISSUE #894: Transform github repo url into website url
If a website's github repo is https://github.com/barack_obama/2016-01-01-whitehouse then we know the website is https://barack_obama.github.io/2016-01-01-whitehouse

When using the "Import" or "Update from URL" it requires the *website* rather than the *repo*.  However, if given the repo, can't we have AMY transform the URL into a valid website URL?  

Often I get a repo url from an instructor (esp. for self organized) workshops and I manually do that transformation myself.

### PR DESCRIPTION
**Pull request by narayanaditya95**
Fixes #894

### CODE REVIEW

File: workshops/util.py [hunk 1]
```diff
@@ -473,21 +473,21 @@ class WrongWorkshopURL(ValueError):
     (see `generate_url_to_event_index` below)."""
 
     def __str__(self):
-        return ('Event\'s URL doesn\'t match Github website format '
-                '"http://user.github.io/2015-12-08-workshop".')
+        return 'Event\'s URL doesn\'t match Github website or repo format.'
```
Comment (narayanaditya95) [inline workshops/util.py]: Should it be more verbose and list all valid URL patterns?

Comment (chrismedrela) [inline workshops/util.py]: IMO no.

**Comment by chrismedrela** [comment]
@narayanaditya95 please address the minor issue that I raised and, then, it's ready to be merged.

**Comment by narayanaditya95** [comment]
@chrismedrela ping.

**Pull request merged**